buildkit/pg_cron.yaml

```diff
@@ -7,8 +7,11 @@ source: https://github.com/citusdata/pg_cron/archive/refs/tags/v1.6.2.tar.gz
 description: Run periodic jobs in PostgreSQL.
 licence: PostgreSQL
 keywords:
-  - cron jobs
-  - periodic jobs
+  - cron
```

Comment:
Why are we breaking up the keywords of `cron jobs` which make more sense than just `cron` or `jobs`

Reply:
@swijckmans Opinions on multi word keywords? Not sure what you think here.

Reply:
Before we had 3 words together with like 30 characters. They were no longer "tags" they were borderline descriptions. Just try keeping it short.

"Shirt" vs "black nike shirt with white logo"

So in the case if cron index or rum index call out "index" because thats what it is. The name of an index is not a keyword thing.

Does that clarify?